Source/Core/Core/AchievementManager.cpp

```diff
@@ -674,111 +247,58 @@ void AchievementManager::DoFrame()
   }
   {
     std::lock_guard lg{m_lock};
-    rc_runtime_do_frame(
-        &m_runtime,
-        [](const rc_runtime_event_t* runtime_event) {
-          GetInstance().AchievementEventHandler(runtime_event);
-        },
-        [](unsigned address, unsigned num_bytes, void* ud) {
-          return static_cast<AchievementManager*>(ud)->MemoryPeeker(address, num_bytes, ud);
-        },
-        this, nullptr);
+    rc_client_do_frame(m_client);
   }
   if (!m_system)
     return;
   time_t current_time = std::time(nullptr);
-  if (difftime(current_time, m_last_ping_time) > 120)
+  if (difftime(current_time, m_last_ping_time) > 10)
```

Comment:
Similar to the other review, it'd be nice if these used `std::chrono` times (in  particular a `steady_clock`) to improve readability and avoid clock changes.  Not sure offhand what time `10` is in.  I guess seconds?